scripts/archive2web.ml

```diff
@@ -103,6 +109,11 @@ let parse_author_name s =
     Str.matched_string s
   else raise (Invalid_argument "author_name")
 
+let obind o f =
```

Comment:
`Option.map` from https://caml.inria.fr/pub/docs/manual-ocaml/libref/Stdlib.Option.html

Reply:
This is since 4.08, hence my comment

Reply:
Yeah, I used that initially. Then I hit some issues with our CI not using 4.08, I fixed it. Then I hit issues with Nix not packaging an 4.08-compatible version of `ppx_deriving_yojson`. I fixed it. Then I hit an issue with the nix channel not being up-to-date yet. Then I thought I'd rather define the function.

And btw, this is `bind`, not `map`, hence the name ;)

Reply:
Hem, so, are you on 4.8? Cause the pr says so. Anyway the changes are trivial, they can be merged I guess

Reply:
> Hem, so, are you on 4.8? Cause the pr says so. Anyway the changes are trivial, they can be merged I guess

Ah sorry! I forgot to update the PR message when I switched back to 4.06, now I understand the confusion.